### ISSUE TITLE
large-v3 The current model is English-only

### ISSUE THREAD

**Issue by drohack** [SYSTRAN/faster-whisper#619]
I'm attempting to transcribe an audio file that's in Japanese using the new large-v3 model. I swear I had this working yesterday, but I don't know what I changed (if anything) that would cause me to get the following error:
`The current model is English-only but the language parameter is set to 'ja'; using 'en' instead.`

It's not able to transcribe the file as the language does not match. Expected outcome it is able to transcribe the Japanese audio.

Looking through the issues I saw this pop up back in November, but it looked like those issues were resolved? So I'm opening this one as I'm not sure what to do about this.

code:
```
from faster_whisper import WhisperModel
...
    # Transcribe with faster_whisper
    model = WhisperModel("large-v3", device="cuda", compute_type="auto", num_workers=5)
    segments, info = model.transcribe(audio=audio_file_path, beam_size=1, language='ja', temperature=0,
                                      word_timestamps=True, condition_on_previous_text=False,
                                      no_speech_threshold=0.1,
                                      )
```

I have the following versions:

- faster-whisper 0.10.0
- torch 2.1.2+cu118
- torchaudio 2.1.2+cu118

This does work with "large-v2". I have not tried using the normal openai-whisper-large-v3 code to test yet

**Comment by drohack** [comment]
Ok yes it only took 25 minutes to transcribe a 22 minute audio file with normal openai/whisper-large-v3 (rather than the 1.10 minutes with faster_whisper). But I was able to transcribe the japanese audio correctly (with the improved transcription) using the default openai/whisper-large-v3.

code:
```
import whisper
...
    model = whisper.load_model("large-v3")  # Change this to your desired model
    transcript = model.transcribe(audio=audio_file_path, language='ja', fp16=False, verbose=False, temperature=0,
                                  word_timestamps=True, condition_on_previous_text=False,
                                  no_speech_threshold=0.1,
                                  )
    segments = transcript['segments']
```

version:

- whisper 1.1.10

So i'm not sure why faster_whisper isn't working.

**Comment by drohack** [comment]
I solved it... Totally my fault. I was testing a different translation tool that downgraded my ctranslate2.

I fixed it by running:
`pip install -U openai-whisper`

This showed me the error:
`ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
argostranslate 1.9.1 requires ctranslate2==3.20.0, but you have ctranslate2 3.23.0 which is incompatible.`

So I also uninstalled argos: `pip3 uninstall argostranslate` (as I don't need it anymore).

So yeah... Make sure your libraries are up to date.
